microsoft/atlas-design
# Dependencies update - June edition

Task: task-[work-item-number]

Link: preview-553

Updating dependencies.

## Testing

1. Visit the site. Everything should load fine.
2. Build succeeds, all checks are passed.
